src/main/java/net/syscon/prison/repository/jpa/model/Offender.java

```diff
@@ -45,9 +45,19 @@
     @Builder.Default
     private String idSourceCode = "SEQ";
 
+    @Column(name = "FIRST_NAME", nullable = true)
+    private String firstName;
+
+    @Column(name = "MIDDLE_NAME", nullable = true)
+    private String middleName;
+
     @Column(name = "LAST_NAME", nullable = false)
     private String lastName;
 
+    @Column(name = "BIRTH_DATE", nullable = true)
+    @CreatedDate
```

Comment:
Did you mean for this `@CreatedDate` annotation here?

Reply:
removed